uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<launch>
+    <arg name="zed_id1"               default="0" />
+
+    <arg name="pose_frame"            default="map" />
+    <arg name="camera_model"          default="0" /> <!-- 0=ZED, 1=ZEDM-->
+    <arg name="serial_number"         default="0" />
+    <arg name="verbose"               default="true" />
+
+    <arg name="resolution"            default="2" /> <!--0=RESOLUTION_HD2K, 1=RESOLUTION_HD1080, 2=RESOLUTION_HD720, 3=RESOLUTION_VGA -->
+    <arg name="frame_rate"            default="30" />
+    <!-- RESOLUTION_HD2K   -> 2208*1242, available framerates: 15 fps.
+         RESOLUTION_HD1080 -> 1920*1080, available framerates: 15, 30 fps.
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <arg name="depth_topic"           default="depth/depth_registered" />
+    <arg name="depth_cam_info_topic"  default="depth/camera_info" />
+    <arg name="laserscan_topic"       default="depth/laserscan"/>
+
+    <arg name="base_frame"            default="zed_camera_center"/>
+    <arg name="laserscan_height"      default="10"/>
+    <arg name="range_min"             default="0.45"/>	
+
+    <!-- First ZED camera -->
+    <group ns="zed1">
+        <include file="$(find zed_wrapper)/launch/zed_camera.launch">
+            <arg name="zed_id"                value="$(arg zed_id1)" />
+
+            <arg name="pose_frame"            value="$(arg pose_frame)" />
+            <arg name="camera_model"          value="$(arg camera_model)" />
+            <arg name="serial_number"         default="$(arg serial_number)" />
+            <arg name="resolution"            default="$(arg resolution)" />
+            <arg name="frame_rate"            default="$(arg frame_rate)" />
+            <arg name="verbose"               default="$(arg verbose)" />
+
+            <arg name="depth_topic"           default="$(arg depth_topic)" />
```

Comment:
I don't think we should append `_topic`, it's redundant - Doesn't make a big difference here, but I don't want to set a precedent

Reply:
I second this. Don't want it to be too verbose

Reply:
I was on my phone before and I guess I misread this, sorry. I think that this doesn't need to be remappable at all, since it's within the namespace the topics can just be `zed1/depth`, `zed1/laserscan`, etc.

With regards to appending `_topic`, I take back what I said - In this case, I still vote we remove this argument all together, but if we are taking topic names as arguments, I think it's okay/preferable for the *argument* to be named `depth_topic`. I misread this before, I meant that I don't think *topics* should have `_topic` appended. Sorry for the confusion.